ulib/FStar.List.fst

```diff
@@ -323,7 +323,7 @@ contains the first [n] elements of [l] and [l2] contains the
 rest. Raises an exception if [l] has fewer than [n] elements. *)
 val splitAt: nat -> list 'a -> ML (list 'a * list 'a)
 let rec splitAt n l =
-  if n = 0 then l, []
+  if n = 0 then [], l
```

Comment:
wow

Reply:
Yeah :)...

Not sure if it's serious, though? Is this always realized by `FStar_List.ml`?